minetest/minetest_game
# Fix missing papyrus in savanna, add a dry dirt version

Papyrus was missing from the dry dirt 'savanna_shore' biome due to
only being placed on dirt. A mistake made when changing savanna
dirt to dry dirt.

Rename existing papyrus schematic to 'papyrus_on_dirt' and only
place in the 'rainforest_swamp' biome.
Add a new papyrus schematic 'papyrus_on_dry_dirt', placed in the
'savanna' biome.

//////////////////////////

![screenshot_20200203_200245](https://user-images.githubusercontent.com/3686677/73687900-dc30b200-46c2-11ea-8229-3e47aa68c3b0.png)

![screenshot_20200203_200656](https://user-images.githubusercontent.com/3686677/73687914-e2bf2980-46c2-11ea-90f8-3840a330b200.png)

Closes #2577

Summary of changes:
* Existing papyrus schematic, which contains dirt nodes, is renamed to 'papyrus_on_dirt'.
* New papyrus schematic 'papyrus_on_dry_dirt' is added, containing dry dirt nodes.
* The existing decoration registration for papyrus is duplicated, and both have their 'name', 'place on', 'biomes' and 'schematic' fields edited.

How to test:
Add these lines to .conf:
```
mg_biome_np_heat = {
    offset      = 100,
    scale       = 0,
    spread      = (1000, 1000, 1000),
    seed        = 5349,
    octaves     = 3,
    persistence = 0.5,
    lacunarity  = 2.0,
    flags       = eased
}
mg_biome_np_humidity = {
    offset      = 55,
    scale       = 10,
    spread      = (1000, 1000, 1000),
    seed        = 842,
    octaves     = 3,
    persistence = 0.5,
    lacunarity  = 2.0,
    flags       = eased
}
```
These settings will make a world only have savanna and rainforest biomes.
Fly / teleport around to check papyrus is present in both biomes (keep in mind it is placed in patches, not everywhere).
